gems/aws-sdk-s3/aws-sdk-s3.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency('aws-sdk-kms', '~> 1')
   spec.add_dependency('aws-sigv4', '~> 1.4')
-  spec.add_dependency('aws-sdk-core', '~> 3', '>= 3.165.0')
+  spec.add_dependency('aws-sdk-core', '~> 3', '>= 3.173.0')
```

Comment:
I thought this would be overwritten by codegen?

Reply:
It would. This is a relic from testing (manual build). I can remove